pymonetdb/sql/pythonize.py

```diff
@@ -89,6 +90,9 @@ def py_timestamptz(data):
     else:
         return datetime.datetime.strptime(dt, '%Y-%m-%d %H:%M:%S') + timezone_delta
 
+def py_sec_interval(data):
```

Comment:
i'm trying to slowly add typing everywhere also, can you add this to this function? What is the expected type of data? probably str.

Reply:
With typing you mean the additional doc strings at the top of the body of Python functions?

Reply:
no `def (data: str) -> datetime.timedelta`